wolfcrypt/src/rsa.c

```diff
@@ -3147,6 +3172,23 @@ static int wc_RsaFunction_ex(const byte* in, word32 inLen, byte* out,
 #else /* !WOLF_CRYPTO_CB_ONLY_RSA */
     SAVE_VECTOR_REGISTERS(return _svr_ret;);
 
+    /* perform software padding here */
+    if(!padded) {
+        in2Len = inLen;
+        in2 = in;
+        if ((type == RSA_PUBLIC_ENCRYPT || type == RSA_PRIVATE_ENCRYPT) && padding->pad_type != WC_RSA_NO_PAD) {
+            in2Len = wc_RsaEncryptSize(key);
+            ret = wc_RsaPad_ex(in, inLen, out, in2Len, padding->pad_value, rng, padding->pad_type,
```

Comment:
```rsa.c
c:\jenkins\workspace\prb-windows-test-v2\wolfcrypt\src\rsa.c(3181): warning C4244: 'function' : conversion from 'int' to 'byte', possible loss of data [C:\jenkins\workspace\PRB-windows-test-v2\wolfssl.vcxproj]
  ripemd.c```

Reply:
Will push shortly

Reply:
Change pad_value member in struct RsaPadding to byte